### ISSUE TITLE
`@sort` deprecation message missing replacement code

### ISSUE THREAD

**Issue by jacobq** [rwjblue/ember-computed-decorators#431]
When I use a line like this in my Ember 3.9.0 app (w/ `ember-decorators` and `ember-decorators-polyfill`) I get a deprecation warning (expected) with apparently missing / incorrect instructions (unexpected).

```js
import { mapBy, sort, uniq } from '@ember-decorators/object/computed';
```

![image](https://user-images.githubusercontent.com/1633459/55812442-a033bd00-5ab0-11e9-95a3-dbf89ef4f6c1.png)

**Comment by jacobq** [comment]
This seems to be coming from 
https://github.com/ember-decorators/ember-decorators/blob/84414eb0c4497d228ecf4622cae2f4120c101d79/packages/object/addon/computed.js#L705
https://github.com/ember-decorators/ember-decorators/blob/84414eb0c4497d228ecf4622cae2f4120c101d79/packages/object/addon/-private/util.js#L65
https://github.com/ember-decorators/ember-decorators/blob/84414eb0c4497d228ecf4622cae2f4120c101d79/packages/object/addon/-private/util.js#L116-L120
https://github.com/ember-decorators/ember-decorators/blob/84414eb0c4497d228ecf4622cae2f4120c101d79/packages/utils/addon/computed.js#L51 
since the calling function didn't provide `importDesc`.

A band-aid could be to check that `importDesc` is truthy before displaying it in the template string so that at least `undefined;` doesn't appear as the suggested replacement code. Better, perhaps `legacyMacroWithMethod` could be updated to pass the needed information through.

**Comment by jacobq** [comment]
Is the suggested replacement code `import { /* legacy macros here */ } from '@ember/object/computed';`?

**Comment by pzuraq** [comment]
Yup, basically all of these should be imported directly from Ember now. We’re working on the latest release, which will update the documentation and remove these methods.
